pages/index.js

```diff
@@ -8,7 +8,7 @@ export default function HomePage() {
       WebViewer(
         {
           path: '/webviewer/lib',
-          initialDoc: '/files/pdftron_about.pdf',
+          initialDoc: "/files/PDFTRON_about.pdf",
```

Comment:
Thanks! Can you just switch this to single quotes to be consistent with the rest of the code?
```suggestion
          initialDoc: '/files/PDFTRON_about.pdf',
```

Reply:
Yeah I can do that no problem.